private/model/api/passes.go

```diff
@@ -7,6 +7,17 @@ import (
 	"strings"
 )
 
+func (a *API) enableGeneratedTypedErrors() {
+	switch a.Metadata.Protocol {
+	case "json":
+	case "rest-json":
```

Comment:
Does it make sense to have this check in load.go, so that it's evident that generated typed errors are only supported for Json, and rest-json protocols.

Alternatively a comment can be added in load.go where we call enableGeneratedTypedErrors to highlight the supported protocols?